src/main/java/seedu/address/ui/InformationDisplay.java

```diff
@@ -59,7 +59,7 @@ public InformationDisplay(Person selectedPerson, String attendance) {
         address.setText(this.person.getAddress().value);
         address.setPrefWidth(150);
         address.setWrapText(true);
-        gender.setText(this.person.getGender().genderOfPerson);
+        //gender.setText(this.person.getGender().genderOfPerson);
```

Comment:
@ChangUo79 remember to implement this in the FXML, otherwise this will be just dead code 😵.